### ISSUE TITLE
Elastic IPs

### ISSUE THREAD

**Issue by Tobeyforce** [djnicholson/ec2-rotate-ip#1]
This package seems excellent, do I understand correctly that this script will automatically switch an IP without a restart being required?
Would a similar approach work with elastic IP's?
I'm looking to find a way to rotate my instance with a pool of 10-15 elastic IP's.

**Comment by djnicholson** [comment]
Yes, the script uses Elastic IPs and does not need a restart.  It always releases the old Elastic IP and requests a new one though, if you wanted to maintain your pool of IPs for later re-use you would need some modifications.
